membersrvc/ca/ca.go

```diff
@@ -33,7 +33,7 @@ import (
 
 	"github.com/hyperledger/fabric/core/crypto/primitives"
 	pb "github.com/hyperledger/fabric/membersrvc/protos"
-	_ "github.com/mattn/go-sqlite3"
+	_ "github.com/mattn/go-sqlite3" // TODO: not use a blank import!
```

Comment:
I hope I addressed the questions/concern, otherwise do let me know?
Even though I personally don't like blank imports (or at least avoid them whenever possible), I have also changed the import comment as follows:

-       _ "github.com/mattn/go-sqlite3" // TODO: not use a blank import!
+       _ "github.com/mattn/go-sqlite3" // TODO: justify this blank import or remove

Please kindly let me know if these address the questions and concerns, or otherwise? Thanks.